tests/test_validation.py

```
@@ -0,0 +1,439 @@
+"""Unit tests for input validation and sanitization."""
+
+import pytest
+import tempfile
+import os
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)